Sofa/Component/Constraint/Lagrangian/Solver/src/sofa/component/constraint/lagrangian/solver/GenericConstraintSolver.h

```diff
@@ -169,7 +172,7 @@ class SOFA_COMPONENT_CONSTRAINT_LAGRANGIAN_SOLVER_API GenericConstraintSolver :
     std::vector<char> constraintCorrectionIsActive; // for each constraint correction, a boolean that is false if the parent node is sleeping
```

Comment:
For the sake of consistency, I guess the change from char to bool should be made here as well, like in GenericConstraintSolver...
```suggestion
    std::vector<bool> constraintCorrectionIsActive; // for each constraint correction, a boolean that is false if the parent node is sleeping
```

Reply:
you're right. And it is another proof that both solvers share common code and need a refactoring